dashboard/src/views/bench/BenchConfig.vue

```diff
@@ -0,0 +1,429 @@
+<template>
```

Comment:
This file is duplicating quite a lot from `SiteConfig.vue`.

Maybe refactor the common component out `JsonConfig.vue` or something like that?

Reply:
My hunch: you'll need this in few more places as we add/expose more backend features.

Reply:
#960